packages/roosterjs-editor-plugins/lib/TabIndent/TabIndent.ts

```diff
@@ -113,4 +143,35 @@ export default class TabIndent implements EditorPlugin {
 
         return false;
     }
+
+    // Check if it is a blockquote event, if it is true, return the blockquote element where the cursor resides
+    // To qualify a blockquote event:
+    // 1. Cursor is in blockquote element
+    // 2. Current block has no content
+    // 3. is keyDown
+    // 4. is Enter
+    // 5. Any of ctrl/meta/alt is not pressed
+    private getBlockQuoteElementFromEvent(event: PluginEvent): Element {
+        if (event.eventType == PluginEventType.KeyDown) {
+            let keyboardEvent = (event as PluginDomEvent).rawEvent as KeyboardEvent;
```

Comment:
You already have keyboardEvent in onPluginEvent, why not just pass it in?

Reply:
Changed.